src/test/java/net/sf/jabref/importer/fileformat/BibtexParserTest.java

```diff
@@ -310,6 +310,40 @@ public void parseRecognizesEntryWhereLastFieldIsFinishedWithComma() throws IOExc
     }
 
     @Test
+    public void parseRecognizesEntryWithAtInField() throws IOException {
+        ParserResult result = BibtexParser.parse(new StringReader("@article{test,author={Ed von T@st}}"));
+
+        List<BibEntry> parsed = result.getDatabase().getEntries();
+        assertEquals(1, parsed.size());
```

Comment:
It is a bit better to compare the parsed list to an expected entry list, i.e.
````
BibEntry expected = new BibEntry("article", "test").withField("author", "bla");
assertEquals(expected as list, parsed);
````